base/applications/cleanmgr/CMakeLists.txt

```
@@ -0,0 +1,20 @@
+remove_definitions(-D_WIN32_WINNT=0x502)
+add_definitions(-D_WIN32_WINNT=0x600)
+
+list(APPEND SOURCE
+    dialog.c
+    main.c
+    mainpage.c
+    optionspage.c
+    util.c
+    stageflagpage.c
```

```suggestion
    stageflagpage.c
    util.c
```
(using alphabetic order).